src/Controller/IndexController.php

```diff
@@ -136,53 +103,14 @@ protected function getIndex(Request $request)
         return $response;
     }
 
-    protected function getAsset(Request $request, string $path, bool $versionedStaticFile): Response
-    {
-        $content = $this->fs->readFile($path);
-        $file = new \SplFileObject($path, 'r');
-        $lastModified = new DateTime();
-        $lastModified->setTimestamp($file->getMTime());
-        $response = new Response();
-        $response = $this->responseFromString($response, $content, $request, $lastModified);
-
-        $extension = $file->getExtension();
-        if ($extension === 'css') {
-            $response->headers->set('Content-Type', 'text/css');
-        }
-        if ($extension === 'js') {
-            $response->headers->set('Content-Type', 'text/javascript');
-        }
-        if ($extension === 'svg') {
-            $response->headers->set('Content-Type', 'image/svg+xml');
-        }
-        if ($extension === 'png') {
-            //PNG files are already compressed and we don't gzip them again
-            $response->headers->remove('Content-Encoding');
-            $response->headers->set('Content-Type', 'image/png');
-        }
-        if ($versionedStaticFile) {
-            //cache for one year
-            $response->setMaxAge(60 * 60 * 24 * 365);
-        } else {
-            // doesn't actually mean don't cache - it means that the server must
-            // check the status and if a 304 is returned it can use the cached version
-            $response->headers->addCacheControlDirective('no-cache');
-        }
-
-        // checks if the file has been modified and if not blanks out the response and sends a 304
-        $response->isNotModified($request);
-
-        return $response;
-    }
-
     /**
      * Extract the path for a frontend file
      * @param $fileName
      * @return bool|string
      */
     protected function getFilePath($fileName)
     {
-        $assetsPath = $this->getParameter('kernel.cache_dir') . UpdateFrontendCommand::FRONTEND_DIRECTORY;
+        $assetsPath = $this->kernelCacheDir . UpdateFrontendCommand::FRONTEND_DIRECTORY;
```

Comment:
pulling this value from a Command constant breaks the boundaries of application structure. feel free to ignore.